lib/CMakeLists.txt

```diff
@@ -131,17 +131,16 @@ endif()
 
 message("Building targets: ${LLVM_TARGETS_TO_BUILD}")
 
-set(LLVM_ENABLE_BINDINGS OFF)
-set(LLVM_ENABLE_OCAMLDOC OFF)
-set(LLVM_ENABLE_PIC OFF)
-set(LLVM_ENABLE_TERMINFO OFF)
-set(LLVM_ENABLE_WARNINGS OFF)
-set(LLVM_ENABLE_Z3_SOLVER OFF)
-set(LLVM_INCLUDE_BENCHMARKS OFF)
-set(LLVM_INCLUDE_TESTS OFF)
-set(LLVM_TOOL_REMARKS_SHLIB_BUILD OFF)
-set(LLVM_ENABLE_ZLIB OFF)
-set(LLVM_ENABLE_ZLIB:STRING OFF)
+set(LLVM_ENABLE_BINDINGS OFF CACHE BOOL "")
+set(LLVM_ENABLE_OCAMLDOC OFF CACHE BOOL "")
+set(LLVM_ENABLE_PIC OFF CACHE BOOL "")
+set(LLVM_ENABLE_TERMINFO OFF CACHE BOOL "")
+set(LLVM_ENABLE_WARNINGS OFF CACHE BOOL "")
+set(LLVM_ENABLE_Z3_SOLVER OFF CACHE BOOL "")
+set(LLVM_INCLUDE_BENCHMARKS OFF CACHE BOOL "")
+set(LLVM_INCLUDE_TESTS OFF CACHE BOOL "")
+set(LLVM_TOOL_REMARKS_SHLIB_BUILD OFF CACHE BOOL "")
+set(LLVM_ENABLE_ZLIB OFF CACHE BOOL "")
```

Comment:
what about adding a comment in the docstring to the effect that we were overwriting the llvm cache entries. This will be visible in the resulting `CMakeCache.txt`. So when someone else is debugging why things aren't working, they might spot those lines here?

Reply:
Sure, suggestion on what to put?

Reply:
e.g. "ponyc overwrite of llvm cache entry"

Reply:
updated.